warehouse/templates/email/password-reset-by-admin/body.html

```diff
@@ -0,0 +1,37 @@
+{#
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ # http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-#}
+{% extends "email/_base/body.html" %}
+
+
+{% block content %}
+<h3>{% trans %}What?{% endtrans %}</h3>
```

Comment:
I realize this got copy/pasted from elsewhere but we probably shouldn't be translating these emails -- they will be localized to the sender (`request.user`, i.e. us), and not the recipient.

Reply:
ok! I'll untranslated this one.

Reply:
hm, end result either way (given that my understanding is _most_ admins are operating in en-us) is an email in US English.

is leaving the translatable strings there better than removing them, in the event we some day offered users to set a language preference in the db... and applied it to emails?

Reply:
Probably an OK assumption to make right now, but might produce confusion if we ever have an admin/moderator that sets their locale to something other than English... maybe better to avoid it? I've been removing these where possible and tracking them in https://github.com/pypi/warehouse/issues/7455

Reply:
https://github.com/pypi/warehouse/pull/16247/commits/0b9a63a992a832a4fa868571067858ca89398497